features/multi-column.yml

```diff
@@ -0,0 +1,19 @@
+name: Multi-column layout
+description: Multi-column layout flows an element's content across one or more vertical content areas in a single row, without affecting the `display` properties of its children.
+spec: https://drafts.csswg.org/css-multicol-1/
+group: layout
+caniuse: multicolumn
```

Comment:
Hmm, this tells a very different support story. I would say multicolumn layout has high support, as in you can use most of it. I'd compute the status from css.properties.columns, and then probably not include the caniuse key since it is so different (and I don't see a way to make it match)